src/server/users/models.js

```diff
@@ -30,42 +33,153 @@ export default class UserManager {
     }
     return false;
   }
-  static getPostKeys(postType, userKey, cb) {
-    const prefix = `${postType}-${STATE.ALIVE}-${userKey}`;
-    fetchPrefix(prefix, (err, keyData) => {
-      cb(err, keyData);
+  static getPostKeys(postType, userKey) {
+    return new Promise((resolve, reject)=>{
```

Comment:
This function is used in savedPostManager and createdPostManager's getPosts function.
These functions are ran asynchronously.
If this function's return value is not Promise object, We have to use many callback and illegible code